Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs

```diff
@@ -548,9 +548,9 @@ private Task<HeartbeatResponse> HeartbeatAsync(HeartbeatRequest request, Cancell
                 errorMessage => new HeartbeatResponse { Header = ResponseHeader.Failure(startTime, errorMessage) });
         }
 
-        private void LogRequestHandling([CallerMemberName]string requestType = "Unknown")
+        private void OperationStarted([CallerMemberName]string requestType = "Unknown")
         {
-            _logger.Debug($"{requestType} request handled by GRPC server.");
+            // TODO: Add counter, but don't trace

```

Comment:
>TODO [](start = 15, length = 4)

is this todo for later? Or intented to do in this pr

Reply:
This todo is for later. But its not that hard to do. So I'll do it.